index.html

```diff
@@ -31,44 +31,44 @@ <h1>My GSoC'24 Journey</h1>
 
         <main>
             <div class="main-content">
-                <p>Ahh, who could&#39;ve thought? I spent my entire summer coding away as part of <a href="summerofcode.withgoogle.com">Google Summer of Code</a>! Feels like the summer just began yesterday.</p>
+                <p>Ahh, who could&#39;ve thought? I spent my entire summer coding away as part of <a href="summerofcode.withgoogle.com" class="effect">Google Summer of Code</a>! Feels like the summer just began yesterday.</p>
```

Comment:
Here I have encountered a bug (ouch)!

I should have put `https://` in front of the link. Now I can fix this, but after I make my commit you would have to rebase your repo which might make things complicated more than it needs to be. You may want to fix this (as it's an easy fix) if you don't want to rebase.